implementations/micrometer-registry-wavefront/src/main/java/io/micrometer/wavefront/WavefrontMeterRegistry.java

```diff
@@ -402,6 +298,14 @@ protected DistributionStatisticConfig defaultHistogramConfig() {
                 .build()
                 .merge(DistributionStatisticConfig.DEFAULT);
     }
+  
+    static String getWavefrontReportingUri(WavefrontConfig wavefrontConfig) {
+        // proxy reporting is now http reporting on newer wavefront proxies.
```

Comment:
Should we mention a minimum version of the proxy in release notes? Is there any concern some users may still be running an older version of the proxy?

Reply:
It would have to be a very old proxy, @basilisk487 care to comment?

Reply:
I believe HTTP support was added in 4.29, which was released in early July of 2018. Since we consider all 4.x and 5.x releases deprecated, it should have us covered.